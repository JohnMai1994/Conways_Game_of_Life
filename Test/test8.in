new 10
init
5 5 5 6 6 5 6 6
-1 -1
print
step
print
step
print
step
print
