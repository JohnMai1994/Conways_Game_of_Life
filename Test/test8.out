__________
__________
__________
__________
__________
_____XX___
_____XX___
__________
__________
__________
__________
__________
__________
__________
__________
_____XX___
_____XX___
__________
__________
__________
__________
__________
__________
__________
__________
_____XX___
_____XX___
__________
__________
__________
__________
__________
__________
__________
__________
_____XX___
_____XX___
__________
__________
__________
